src/app/components/СonfigForm.tsx

```diff
@@ -1,77 +1,78 @@
 import * as React from 'react';
 import {useInput} from '../hooks/inputHook';
 import {FormEvent} from 'react';
+import {File} from '../utils/githubUtils';
 
 export type Config = {
-  repoPath: string;
+  repoName: string;
   token: string;
-  committerName: string;
-  committerEmail: string;
+  ownerName: string;
   headBranch: string;
   baseBranch: string;
 };
 
+type NetworkRequestMessage = {
+  type: 'NETWORK_REQUEST',
+  content: File[],
+  config: Config,
+  fileName: string
+}
+
+type GithubConfigMessage = {
+  type: 'GITHUB_CONFIG',
+  config: Config;
+}
```

Comment:
Некритично
Тут можно вот так было, чтобы не убирать константы, и `src/app/components/App.tsx` тут их использовать

```suggestion
const NETWORK_REQUEST = 'NETWORK_REQUEST';
const GITHUB_CONFIG = 'GITHUB_CONFIG';

type NetworkRequestMessage = {
  type: typeof NETWORK_REQUEST,
  content: File[],
  config: Config,
  fileName: string
}

type GithubConfigMessage = {
  type: typeof GITHUB_CONFIG,
  config: Config;
}
```